code/datums/view.dm

```diff
@@ -116,7 +116,9 @@
 	//Ready for this one?
 	setTo(radius)
 
-/proc/getScreenSize(widescreen) //IMPORTANT: If widescreen toggle preference gets ported, several uses of this proc need to be changed from FALSE to the player pref
+/proc/getScreenSize(widescreen, mob/M) //IMPORTANT: If widescreen toggle preference gets ported, several uses of this proc need to be changed from FALSE to the player pref
+	if(CONFIG_GET(flag/menu_square_view) && M && istype(M, /mob/dead/new_player))
+		return "15x15"	//Return 15x15 for new players because we have normal sized menu screens
 	if(widescreen)
 		return CONFIG_GET(string/default_view)
-	return "15x15" //IMPORTANT: If widescreen toggle preference gets ported, this needs to be set to the square view config
+	return "17x15" //IMPORTANT: If widescreen toggle preference gets ported, this needs to be set to the square view config
```

Comment:
No. This should be kept at 15x15, and you should be handling this as a config option.

Reply:
Good point, this whole proc needs redoing since the widescreen var is always set to false